### PR TITLE
Add check for amd + iommu=pt

### PR DESCRIPTION
**Pull request by dosaboy**
Resolves: #319

### CODE REVIEW

File: tests/unit/test_system.py [hunk 1]
```diff
@@ -156,3 +164,41 @@ def test_unattended_upgrades(self):
...
+    @mock.patch('hotsos.core.ycheck.engine.YDefsLoader._is_def',
+                new=utils.is_def_filter('amd_iommu_pt.yaml'))
+    def test_amd_iommu_pt_pass(self, mock_cli, mock_kernel):
+        mock_cli.return_value = mock.MagicMock()
+        mock_cli.return_value.lscpu.return_value = \
+            FAKE_AMD_LSCPU.splitlines(keepends=True)
+
+        mock_kernel.return_value = mock.MagicMock()
+        mock_kernel.return_value.boot_parameters = \
+            ['intel_iommu=on', 'iommu=pt']
```
Comment (gerald-yang-tw) [inline tests/unit/test_system.py:200]: does this mean both intel_iommu and iommu must be set to on and pt?
I am only sure for AMD EPYC series CPU, iommu must be set to pt, but not sure about intel_iommu

Comment (dosaboy) [reply tests/unit/test_system.py:200]: hi @gerald-yang-tw no only iommu=pt is needed:

  iommu_not_pt:
    property:
      path: hotsos.core.plugins.kernel.KernelBase.boot_parameters
      ops: [[contains, 'iommu=pt'], [not_]]

I just put both to test for false positives.

Comment (gerald-yang-tw) [reply tests/unit/test_system.py:200]: Got it, thanks Ed

**Comment by gerald-yang-tw** [comment]
LGTM

**Pull request merged**